src/purify.js

```diff
@@ -7,9 +7,8 @@
         define(function(){ return factory(root); });
     } else if (typeof module !== 'undefined') {
         module.exports = factory(root);
-    } else {
-        root.DOMPurify = factory(root);
     }
+    root.DOMPurify = factory(root);
```

Comment:
I don't know what the issue is with Electron. Can you please elaborate a bit more?